tools/flang2/flang2exe/ll_write.cpp

```diff
@@ -1033,6 +1033,18 @@ static const MDTemplate Tmpl_DIModule[] = {
   //{ "isysroot",               StringField, FlgOptional }
 };
 
+static const MDTemplate Tmpl_DIModule_11[] = {
+  { "DIModule", TF, 5 },
+  { "tag",                      DWTagField, FlgHidden },
+  { "scope",                    NodeField },
+  { "name",                     StringField },
+  { "file",                     NodeField },
+  { "line",                     UnsignedField }
+  //,{ "configMacros",          StringField, FlgOptional },
```

Comment:
NIT: Modern C/C++ standards allow a trailing comma on the last item in an initializer list. Please move this comma back to the previous line, and add one to the end of line 1045 below as well.